src/chrome/content/rules/Rebateaccess.com.xml

```diff
@@ -0,0 +1,31 @@
+<!--
+	Problematic domains:
+
+		- ^ ¹
+		- www ¹
+		- www.americanstandard ²
+		- www.jabrapartner ²
+		- www.pantone ²
+		- www.plantronics ²
+
+	¹: Refused
+	²: Mismatch
+
+	It would seem like all first level subdomains work, but that
+	all others fail; if so, a wildcard rule would be more appropriate,
+	but it is kept like this to be on the safe side.
+-->
+<ruleset name="Rebateaccess.com">
+
+	<target host="americanstandard.rebateaccess.com" />
+	<target host="chat.rebateaccess.com" />
+	<target host="coolermaster.rebateaccess.com" />
+	<target host="hamiltonbeach.rebateaccess.com" />
+	<target host="jabrapartner.rebateaccess.com" />
+	<target host="newegg.rebateaccess.com" />
+	<target host="pantone.rebateaccess.com" />
+	<target host="plantronics.rebateaccess.com" />
```

Comment:
lepa.rebateaccess.com
enermax.rebateaccess.com
paloalto.rebateaccess.com
zeiss.rebateaccess.com
zeissphoto.rebateaccess.com
manfrotto.rebateaccess.com
olympus.rebateaccess.com
barnesbullets.rebateaccess.com
kaspersky.rebateaccess.com
goldeagle.rebateaccess.com
lasermax.rebateaccess.com
remington.rebateaccess.com
marlinfirearms.rebateaccess.com
icom.rebateaccess.com
bushmaster.rebateaccess.com
para.rebateaccess.com
sportsmanswarehouse.rebateaccess.com
aac.rebateaccess.com
...

So far all of them worked for me.